### ISSUE TITLE
Only works on 1 file when screen is split

### ISSUE THREAD

**Issue by Tunscopi** [amos402/vscode-scope-bar#14]
Hello and thanks for this nice extension!
Is it possible to enable this for both screens when the screen is split?

**Comment by amos402** [comment]
Since all other information on status bar refered to the activated document, showing other files' information may being some confuse, I think it's not a good idea to do that.

**Comment by Tunscopi** [comment]
Yes, poor word choice from me. I am  referring to the active document. What I'd observed was that when I switch to the other document (other split screen) it still shows the last clicked method signature from the previous document/screen. I would think following through with the active document would be ideal

**Comment by amos402** [comment]
It should be change to the active document as you said. If not, could you describe how to reproduce it?

**Comment by Tunscopi** [comment]
Yea, this is weird. I have painstakingly tried to follow what's going on but failed so far.
In summary, it seems like it works perfectly 90% of the time. I switch between the split screen files and click in methods and it displays the right method or function name.

Quite often though, it will do the ff:
- When I have the same file open in both splits, will just continue showing a particular method name even if I click elsewhere in either file
- With 2 splits, only update with 1 split
- Completely not work on either.

I wonder if this is due to the size of the project I'm working on which is quite massive with thousands and thousands of files.
Well, I hope to gather clearer info regarding the oddities and report back.
